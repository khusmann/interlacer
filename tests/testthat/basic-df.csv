a,b,c,d
REASON_1,7,0.181526642525569,"m"
TRUE,REASON_2,0.833227441413328,"z"
TRUE,8,0.926790483295918,"r"
REASON_3,10,0.375270307529718,"REASON_3"
