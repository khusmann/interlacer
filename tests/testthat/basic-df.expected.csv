a,.a.,b,.b.,c,.c.,d,.d.
NA,REASON_1,7,NA,0.181526642525569,NA,m,NA
TRUE,NA,NA,REASON_2,0.833227441413328,NA,z,NA
TRUE,NA,8,NA,0.926790483295918,NA,r,NA
NA,REASON_3,10,NA,0.375270307529718,NA,NA,REASON_3
